src/views/Home/index.tsx

```diff
@@ -85,11 +115,31 @@ const HomeView = (): JSX.Element => {
       const response = await getPrefabs(query)
 
       if (response.data && response.data.success) {
-        const categories = getCategories(response.data.data)
+        setCategories(getCategories(response.data.data))
+
+        if (selectedValues.length > 0) {
+          const selectedPacks = selectedValues.map(x => x['value'] as string);
+          const filteredPacks: PrefabsWrapper = {};
+          const remainingPacks: PrefabsWrapper = {};
+
+          Object.values(response.data.data).
```

Comment:
not sure if this is an array, or maybe you could turn it into an array, but the code could be a bit more lightweight I think, something along this:

`const filtered = packs.filter(p => selectedPacks.some(s => p.pack.identifier === s.pack.identifier))`

`const combined = { ...filtered, ...packs }`

Reply:
I believe that would generate duplicates, the packs.filter would retrieve the proper packs but the packs variable itself would remain as is thus causing duplication